code/datums/wires/wires.dm

```diff
@@ -1,119 +1,37 @@
-var/list/wire_colors = list( // http://www.crockford.com/wrrrld/color.html
-	"aliceblue",
-	"antiquewhite",
+var/list/wire_colors = list(
 	"aqua",
-	"aquamarine",
 	"beige",
-	"blanchedalmond",
 	"blue",
-	"blueviolet",
 	"brown",
-	"burlywood",
-	"cadetblue",
-	"chartreuse",
-	"chocolate",
-	"coral",
-	"cornflowerblue",
-	"cornsilk",
 	"crimson",
 	"cyan",
-	"deeppink",
-	"deepskyblue",
-	"dimgray",
-	"dodgerblue",
-	"firebrick",
-	"floralwhite",
-	"forestgreen",
 	"fuchsia",
-	"gainsboro",
-	"ghostwhite",
 	"gold",
-	"goldenrod",
-	"gray",
+	"grey",
 	"green",
-	"greenyellow",
-	"honeydew",
 	"hotpink",
-	"indianred",
 	"ivory",
 	"khaki",
 	"lavender",
-	"lavenderblush",
-	"lawngreen",
-	"lemonchiffon",
-	"lightblue",
-	"lightcoral",
-	"lightcyan",
-	"lightgoldenrodyellow",
-	"lightgray",
 	"lightgreen",
 	"lightpink",
-	"lightsalmon",
-	"lightseagreen",
-	"lightskyblue",
-	"lightslategray",
-	"lightsteelblue",
-	"lightyellow",
 	"lime",
 	"limegreen",
-	"linen",
 	"magenta",
-	"maroon",
-	"mediumaquamarine",
-	"mediumblue",
-	"mediumorchid",
-	"mediumpurple",
-	"mediumseagreen",
-	"mediumslateblue",
-	"mediumspringgreen",
-	"mediumturquoise",
-	"mediumvioletred",
-	"mintcream",
-	"mistyrose",
-	"moccasin",
-	"navajowhite",
-	"oldlace",
 	"olive",
-	"olivedrab",
 	"orange",
-	"orangered",
-	"orchid",
-	"palegoldenrod",
-	"palegreen",
-	"paleturquoise",
-	"palevioletred",
-	"papayawhip",
-	"peachpuff",
-	"peru",
 	"pink",
-	"plum",
-	"powderblue",
 	"purple",
 	"red",
-	"rosybrown",
-	"royalblue",
-	"saddlebrown",
 	"salmon",
-	"sandybrown",
-	"seagreen",
-	"seashell",
-	"sienna",

```

Comment:
this is a legitimate colour, just as valid as some of the others you've left intact.